docs/source/tutorials.rst

```diff
@@ -5,17 +5,30 @@ Tutorials
 
 
 Here a set of examples on how to use different MyoSuite models and non-stationarities.
-Jupyter-Notebooks can be found `here <https://github.com/facebookresearch/myosuite/tree/main/docs/source/tutorials>`__
+It is highly recommended to read through the `OpenAI Gym API <https://gymnasium.farama.org/>`__ before you start.
 
 * :ref:`run_myosuite`
 * :ref:`run_visualize_index_movements`
 * :ref:`run_trained_policy`
-* :ref:`test_muscle_fatigue`
-* :ref:`test_sarcopenia`
-* :ref:`test_tendon_transfer`
-* :ref:`resume_training`
-* :ref:`load_deprl_baseline`
-* :ref:`load_MyoReflex_baseline`
+* :ref:`advanced_muscle_conditions`
+    * :ref:`test_muscle_fatigue`
+    * :ref:`test_sarcopenia`
+    * :ref:`test_tendon_transfer`
+    * :ref:`exoskeleton`
+* :ref:`use_reinforcement_learning`
+    * :ref:`resume_training`
+    * :ref:`load_deprl_baseline`
+    * :ref:`load_MyoReflex_baseline`
+* :ref:`customizing_tasks`
+
+.. _jupyter_notebook:
+
+Tutorials on Jupyter-Notebook
+========================================
+Please refer to our tutorials on Jupyter-Notebook `here <https://github.com/facebookresearch/myosuite/tree/main/docs/source/tutorials>`__
+
+There are also tutorials for our ICRA workshops: `ICRA-2023 <https://colab.research.google.com/drive/1zFuNLsrmx42vT4oV8RbnEWtkSJ1xajEo>`__, `ICRA-2024 <https://colab.research.google.com/drive/1JwxE7o6Z3bqCT4ewELacJ-Z1SV8xFhKK#scrollTo=QDppGIzHB9Zu>`__
```

Comment:
@raku-slyu - add a short summary of what the tutorial covers

Reply:
included a short summary for both ICRA workshops